### ISSUE TITLE
About Evaluation

### ISSUE THREAD

**Issue by fukien** [samet-akcay/ganomaly#37]
Hi, thanks for your paper and your codes.

I've just read the script evaluate.py. In evaluation, you chose three metrics to evaluate the classification performance.
When evaluating auc or average_precision_score, you just use sklearn to calculate the metrics. However, when testing the f-score, you specifically set up a threshold to determine it's positive or negative. I think the result the f1_score is contradictory to the result of auc or average_precision. Could you please explain why you set up the threshold in f-score why you set up the threshold as 0.2 (intuitively speaking, different datasets should be set up different thresholds)?

**Comment by samet-akcay** [comment]
Hi @fukien, we've included f1_score code but never used in the paper. The reason for writing is that some papers prefer to use f1-score metric. We've set the threshold to 0.2, but as you pointed out, that threshold value is data-specific and might vary when the model is trained on different datasets. The ideal use would be to have the threshold variable as an argument within the function definition. I'll try to fix it. Thanks for spotting it.
